src/components/CvssCalculator/CvssCalculator.vue

```diff
@@ -88,24 +95,55 @@ function highlightFactorValue(factor: null | string) {
     @paste="handlePaste"
   >
     <div class="osim-input mb-2">
-      <label class="label-group row">
-        <span class="form-label col-3">
-          RH CVSSv3
+      <label class="label-group row" aria-role="red-hat-cvss">
+        <span class="form-label col-3 pe-1">
+          <img
+            :src="RedHatIconSvg"
+            alt="Red Hat Logo"
+            width="24px"
+            class="me-2"
+          />
+          CVSS
+          <select
+            v-model="cvssVersion"
+            class="ms-2"
+          >
+            <option
+              v-for="(version, index) in CvssVersions"
+              :key="index"
+              :value="version"
+              :selected="cvssVersion === version"
+            >
+              {{ CvssVersionDisplayMap[version] }}
+            </option>
+          </select>
+          <span class="bg-light rounded ms-2 px-2" title="Synchronize factors between CVSS3 and CVSS4">
+            <i
+              class="bi bi-repeat me-1"
+              style="cursor: help; height: 24px; width: 24px; stroke: black; vertical-align: middle;"
+            ></i>
+            <input
+              v-model="shouldSyncVectors"
+              type="checkbox"
+              class="form-check-input"
+            />
+          </span>
         </span>
         <div class="input-wrapper col">
           <CvssVectorInput
             ref="cvssVectorInput"
-            :cvssVector="cvssVector"
-            :cvssScore="cvssScore"
-            :cvssFactors="cvssFactors"
+            :cvss4Vector="cvss4Vector"
+            :cvss3Factors="cvss3Factors"
```

Comment:
Personal taste suggestion (also applies for other cases)
```suggestion
            :cvss4Vector
            :cvss3Factors
```